CMakeLists.txt

```diff
@@ -108,6 +108,14 @@ endif()
 
 
 add_subdirectory(third_party EXCLUDE_FROM_ALL)
+target_compile_definitions(ot-config INTERFACE
```

Comment:
Any reason to put this here? Should these updates to `ot-config` be in `third_party/openthread/CMakeLists.txt`?